### ISSUE TITLE
Storybook installation in an ember-addon fails [Bug]

### ISSUE THREAD

**Issue by fpauser** [storybookjs/ember-cli-storybook#55]
### Describe the bug

Running `ember install @storybook/ember-cli-storybook` in a fresh ember-addon fails:

```
The `ember generate <entity-name>` command requires an entity name to be specified. For more details, use `ember help`.
```

### Steps to reproduce the behavior

1. Generate a fresh ember-addon `ember addon dumy-addon`
2. Cd into dummy-addon `cd dummy-addon`
3. Install '@storybook/ember-cli-storybook' via `ember install @storybook/ember-cli-storybook --yarn`
```
$ ember install @storybook/ember-cli-storybook

🚧  Installing packages... This might take a couple of minutes.
Yarn: Installed @storybook/ember-cli-storybook
installing ember-cli-storybook
The `ember generate <entity-name>` command requires an entity name to be specified. For more details, use `ember help`.
```

### Environment
os: linux x64 (Fedora 33)
node: 14.7.0
yarn: 1.22.5
ember-cli: 3.22.0

**Comment by gossi** [comment]
That is a known issue that so far nobody could answer how to solve it 🤷 

If you are affected by this, you can manually run:

```
ember g @storybook/ember-cli-storybook
```

which will scaffold your storybook installation.

**Comment by gabrielcsapo** [comment]
Looking into fixing this, this weekend.

**Comment by usamahamed** [comment]
BTW: i already tried this command `ember g @storybook/ember-cli-storybook` and i got this error 

```
Unknown blueprint: @storybook/ember-cli-storybook

```

**Comment by gabrielcsapo** [comment]
Fixed in 0.4.0

<img width="747" alt="Screen Shot 2021-04-09 at 6 17 58 PM" src="https://user-images.githubusercontent.com/1854811/114253665-ebaf2900-995f-11eb-8629-3005350ceac2.png">
